recipes/indexed_gzip/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "indexed_gzip" %}
+{% set version = "0.8.2" %}
+{% set sha256 = "77f7bcfe4a5d48ec81b720e4bb6c4813be28eb4db7bac9ab5766cd1bc9369c26" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
```

Comment:
please add ` {{ compiler('c') }}` as well.  when you do you can remove the various cflags and ldflags exports